app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,34 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:id="@+id/layout_search">
+
+        <EditText android:id="@+id/etSearchText"
+            android:layout_weight="1"
```

Comment:
layout_weight will be used only if the parent in LinearLayout and has weightSum defined. Otherwise, use match parent as width